### ISSUE TITLE
Ele, ACF Color picker & Elementer post templates

### ISSUE THREAD

**Issue by tore0259** [dudaster/ele-custom-skin#584]
Hi,

When creating an Ele loop with Elementor dynamic post fields - I have different background colors for different pages using the same loop template - however the background color remains the default option for the loop template no matter what the single pages color picker field shows.
When using just the template w/o the Elementor dynamic post, the colors changes according to the AFC color picker settings of the current page.

Do you have any solutions to fix this issue?

Thank you,
Tore

**Comment by dudaster** [comment]
is there a way you can choose a fallback value for it?

**Comment by tore0259** [comment]
Yes - it is actually the fallback option, not the default option it shows.

**Comment by dudaster** [comment]
can you provide an URL so I can inspect element. Tried on my part and it is working fine.

**Comment by tore0259** [comment]
Yes sure.
http://www.dpfudvikling.dk/ny-i-padel/

The red color is the color picker value of that page. The blue color is the fallback that don't get switched in this case. (Which is an ele loop and then put in the elementor post widget)

**Comment by dudaster** [comment]
I see only one has the acf color picker... Can you make more than one to show?

**Comment by tore0259** [comment]
I dont know if i understand you correctly. The loops used on that page has its column color by ACF color picker.
